Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3400,14 +3400,15 @@ def create_events_search(client: Client,
                          ) -> str:
     additional_where = ''' AND LOGSOURCETYPENAME(devicetype) = 'Custom Rule Engine' ''' \
         if fetch_mode == FetchMode.correlations_events_only.value else ''
+    now: int = int(datetime.now().timestamp() * 1000)
```

Comment:
```suggestion
    now_millis: int = int(datetime.now().timestamp() * 1000)
```
or `now_milliseconds`